src/editors/data/redux/thunkActions/app.js

```diff
@@ -44,9 +44,53 @@ export const saveBlock = ({ content, returnToUnit }) => (dispatch) => {
   }));
 };
 
-export const fetchImages = ({ onSuccess }) => () => {
+export const fetchImages = ({ onSuccess, stopLoading }) => () => {
   // get images
-  onSuccess(mockData.mockImageData);
+  const processedData = Array.from(mockData.mockImageData, el => {
+    const sortDate = new Date(el.dateAdded.replace(' at', '')).getTime();
```

Comment:
Ahh do we want them to arrive sorted by date by default? we can just do that with the api call to make things easier?